README.md

```diff
@@ -29,115 +29,68 @@ rule find_exceptions: trojan
-```
-
-Create a new database:
-
-```
-./ursadb/build/ursadb_new ~/db.ursa
-```
+## Next steps
 
-And start everything manually (systemd services recommended here):
+After you start the system, you should index some files. Use ursadb-cli:
 
 ```
-project_dir/mquery/src$ flask run  # web server
-project_dir/mquery/src$ python3 daemon.py  # job daemon (can be scaled horisontally)
-project_dir/ursadb/build$ ./ursadb ~/db.ursa  # backend database
+sudo docker-compose run ursadb-cli tcp://ursadb:9281
+index "/mnt/samples";  # /mnt/samples refers to ./samples in the repo
 ```
 
-Web interface should be available at `http://localhost:5000`.
+The command will track the progress.
+Wait until it's finished (this can take a while).
 
-## Quick start (kubernetes)
-
-Dealing with Kubernetes is never quick. If you're up for a challenge, take a look
-at the `./deploy/k8s` directory.
-
-## Next steps
-
-After you start the system, you should index some files.
-
-Right now it can only be done with ursadb-cli tool. Start it:
-
-- for bare metal setup: `python3 ursadb-cli/ursaclient.py`
-- for docker: `sudo docker-compose run ursadb-cli tcp://ursadb:9281`
-
-Enter the `index` command (for more complicated options, see ursadb docs):
-
-- for bare metal setup `index "/path/to/your/malware/samples";`
-- for docker: `index "/mnt/samples";` (remember, this is mount configured in `docker-compose.yml`)
-
-The command should print the progress. Wait until it's finished (this can take a while).
-
-Now your files should be searchable - try with the following yara rule (or any other):
+Now your files should be searchable - try with the following Yara rule (or any other):
 
 ```
-rule emotet4_basic: trojan
+rule emotet4_basic
 {
     meta:
         author = "cert.pl"
     strings:
-        $emotet4_rsa_public = { 8d ?? ?? 5? 8d ?? ?? 5? 6a 00 68 00 80 00 00 ff 35 [4] ff 35 [4] 6a 13 68 01 00 01 00 ff 15 [4] 85 }
+        $emotet4_rsa_public = {
+            8d ?? ?? 5? 8d ?? ?? 5? 6a 00 68 00 80 00 00 ff 35 [4] ff
+            35 [4] 6a 13 68 01 00 01 00 ff 15 [4] 85
+        }
         $emotet4_cnc_list = { 39 ?? ?5 [4] 0f 44 ?? (FF | A3)}
     condition:
         all of them
 }
 ```
 
-## Contributing
+## Learn more
 
-If you want to contribute, ensure that your PR passes through the CI pipeline. Ideally:
+See [internals.md](./docs/internals.md) to learn about:
+
+ - How mquery works on a high level.
+ - Known limitations and design decisions.
+ - How to create efficient yara rules.
+
+## Contributing
 
- - check your code with `flake8`
- - autoformat your python code with `black`
- - autoformat your html/js/jsx with `prettier --tab-width=4`
+If you want to contribute, see [CONTRIBUTING.md](./CONTRIBUTING.md).
 
 ## Contact
 
-In case of any questions, feel free to contact:
+If you have any problems, bugs or feature requests related to mquery, you're
+encouraged to create a GitHub issue. If you have other questions or want to
+contact the developers directly, you can email:
```

Comment:
Right now mwdb slack is a pretty curated community (limited to a subset of mwdb users). We could consider opening it more, but we probably don't want to create yet another malware slack (there are dozens of those already) :thinking:. 

I know that email is a pretty poor communication channel, but until we grow more I think we'll keep email and Github as primary methods of getting in touch.

Especially since the bigger users that I know of are on slack anyway :thinking: